OpenRA.Mods.Common/Traits/Palettes/PaletteFromGrayscale.cs

```diff
@@ -0,0 +1,58 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
```

Comment:
2022

Reply:
Copied the palettefromrgba. Needs to be changed there too. Different pr?

Reply:
I don't see why, PaletteFromRGBA might have been touched last in 2021, but this was added in 2022.

Reply:
Setting all headers sounds like a good idea.

Reply:
Oh my bad, it's an issue with commits, first commit adds the header and the second one updates it to 2022